cmd/placement-translator/main.go

```diff
@@ -187,12 +176,12 @@ func main() {
 	kubeClusterInformerFactory := kcpkubeinformers.NewSharedInformerFactory(kubeClusterClient, 0)
 	nsClusterPreInformer := kubeClusterInformerFactory.Core().V1().Namespaces()
 
-	edgeInformerFactory := emcinformers.NewSharedInformerFactoryWithOptions(edgeViewClusterClientset, resyncPeriod)
-	epClusterPreInformer := edgeInformerFactory.Edge().V2alpha1().EdgePlacements()
-	spsClusterPreInformer := edgeInformerFactory.Edge().V2alpha1().SinglePlacementSlices()
-	syncfgClusterPreInformer := edgeInformerFactory.Edge().V2alpha1().SyncerConfigs()
-	customizerClusterPreInformer := edgeInformerFactory.Edge().V2alpha1().Customizers()
-	var _ edgev1a1informers.SinglePlacementSliceClusterInformer = spsClusterPreInformer
+	edgeInformerFactory := emcinformers.NewSharedScopedInformerFactoryWithOptions(edgeClientset, resyncPeriod)
+	epPreInformer := edgeInformerFactory.Edge().V2alpha1().EdgePlacements()
+	spsPreInformer := edgeInformerFactory.Edge().V2alpha1().SinglePlacementSlices()
+	syncfgPreInformer := edgeInformerFactory.Edge().V2alpha1().SyncerConfigs()
+	customizerPreInformer := edgeInformerFactory.Edge().V2alpha1().Customizers()
+	var _ edgev1a1informers.SinglePlacementSliceInformer = spsPreInformer
```

Comment:
This line should have been deleted months ago, it was just a temporary thing to help write the code.

Reply:
Removed